src/quo2/components/navigation/page_nav.cljs

```diff
@@ -5,7 +5,8 @@
             [quo2.components.icon :as icons]
             [quo2.components.markdown.text :as text]
             [quo2.foundations.colors :as colors]
-            [react-native.core :as rn]))
+            [react-native.core :as rn]
+            [status-im.utils.platform :as platform]))
```

Comment:
please don't use code from status_im, this functionality should already be somewhere better in the codebase -> I believe it was migrated to react-native.platform

Reply:
this is actually i mistake I will correct that